### PR TITLE
Add records for missing salesforce types

### PR DESCRIPTION
**Pull request by pramodya1994**
## Purpose
Add records for missing salesforce types

### CODE REVIEW

File: src/sfdc46/salesforce_endpoint.bal [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 import ballerina/http;
 import ballerina/oauth2;
+// import ballerina/log;
```
Comment (Maninda) [inline src/sfdc46/salesforce_endpoint.bal]:
```suggestion
```

File: src/sfdc46/salesforce_types.bal [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    string label;
+    map<string> urls;
+    json...;
+|};
+
+public type SObjectBasicInfo record {|
+    SObjectMetaData objectDescribe;
+    json...;
+|};
+
```
Comment (Maninda) [inline src/sfdc46/salesforce_types.bal]:
```suggestion


```

Comment (pramodya1994) [reply src/sfdc46/salesforce_types.bal]: Done

File: src/sfdc46/salesforce_endpoint.bal (hunk 1 above)
Comment (pramodya1994) [reply src/sfdc46/salesforce_endpoint.bal]: Done

**Pull request merged**